azure-ts-appservice/README.md

```diff
@@ -50,7 +49,11 @@ with App Service.
 
     ```
     $ pulumi stack output endpoint
-    https://azpulumi-as0ef47193.azurewebsites.net
+    https://azpulumi-as0ef47193.azurewebsites.net (you will get a different url)
```

Comment:
Don't think we have to add this here (we have similar many other places as well).